src/test/java/nextstep/subway/line/domain/SectionsTest.java

```diff
@@ -15,23 +13,23 @@
 
 class SectionsTest {
 	private Sections sections;
-	private Station station1;
-	private Station station2;
-	private Station station3;
-	private Station station4;
-	private Station station5;
+	private Station stationDuJeong;
```

Comment:
```suggestion
	private Station 두정역;
```

테스트 코드에서 변수명을 한글로 써보는 건 어떨까요?
좀더 의미 파악하기 쉽지 않을까요?